portfolio/src/main/webapp/images.html

```diff
@@ -11,11 +11,11 @@
     <div id="content">
       <h1>Chloe's Image Gallery</h1>
       <div class="image-thumbnails">
-        <a href="images/IMG_1.jpeg"><img src="images/IMG_1.jpeg" alt="Dog 1"/></a>
-        <a href="images/IMG_2.jpeg"><img src="images/IMG_2.jpeg" alt="Dog 2"/></a>
-        <a href="images/IMG_3.jpeg"><img src="images/IMG_3.jpeg" alt="Dog 3"/></a>
-        <a href="images/IMG_4.jpeg"><img src="images/IMG_4.jpeg" alt="Dog 4"/></a>
-        <a href="images/IMG_5.jpeg"><img src="images/IMG_5.jpeg" alt="Dog 5"/></a>
+        <a href="images/IMG_1.jpeg"><img src="images/IMG_1.jpeg" alt="Dog 1" /></a>
+        <a href="images/IMG_2.jpeg"><img src="images/IMG_2.jpeg" alt="Dog 2" /></a>
+        <a href="images/IMG_3.jpeg"><img src="images/IMG_3.jpeg" alt="Dog 3" /></a>
+        <a href="images/IMG_4.jpeg"><img src="images/IMG_4.jpeg" alt="Dog 4" /></a>
+        <a href="images/IMG_5.jpeg"><img src="images/IMG_5.jpeg" alt="Dog 5" /></a>
```

Comment:
You shouldn't have both a self-closing tag (`<a ... />`) and a closing tag (`</a>`). In HTML5, `a` elements should not be self-closing: https://html.spec.whatwg.org/multipage/syntax.html#syntax-tags

Reply:
I think the self closing tag (/>) at the end of the line is for the <img> element and not the a element.

Reply:
Ah, you're right and I was absolutely wrong! My bad, not sure how I misread that.